maltd-frontend/package-lock.json

```diff
@@ -11687,7 +11686,6 @@
       "version": "9.0.0",
       "resolved": "https://registry.npmjs.org/keycloak-js/-/keycloak-js-9.0.0.tgz",
       "integrity": "sha512-YvxPWKScIrSWCdxfuibTN0HauscsnKJBFhjXk7WAHmcxuEMyR1wRmSjwfxErWlHknJPLNDON5yrK/FDv0CiXlw==",
-      "dev": true,
```

Comment:
not too sure about these changes either....

Reply:
This seems like a reflection of .env file removal. I did not make any changes.